### PR TITLE
Add PolicyEngine to the `Powered by OpenFisca` section

### LINKED ISSUE #94: Add PolicyEngine to the `Powered by OpenFisca` section
Would you be open to adding https://policyengine.org to https://openfisca.org/en/#products?

@nikhilwoodruff fyi

### PR DESCRIPTION
**Pull request by MaxGhenis**
Fixes #94

I translated to French with Google Translate, so suggestions welcome :)

cc @nikhilwoodruff

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
Thanks - to add an image, do we just need to add one with the same name in `static/img/showcase`?

**Comment by clementbiron** [comment]
Thank you for this PR! 

It would be helpful if you could add the image to this PR and yes @nikhilwoodruff it should be added to the folder `static/img/showcase`. It should also be in `png`, in 1280×720px size and and name itself with the [urlize](https://gohugo.io/functions/urlize/) `title`.

File: data/showcase/policyengine_uk.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: PolicyEngine UK
+description:
+  fr: L'application Web PolicyEngine UK utilise un fork d'OpenFisca pour permettre à quiconque d'estimer ses impôts et ses avantages, et d'estimer l'impact des réformes des politiques personnalisées.
```
Comment (MattiSG) [inline data/showcase/policyengine_uk.yml:3]:
```suggestion
  fr: PolicyEngine UK utilise un fork d'OpenFisca pour permettre à quiconque d'estimer ses impôts et aides sociales, ainsi que l'impact de réformes personnalisées.
```

File: data/showcase/policyengine_us.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+title: PolicyEngine US
+description:
+  fr: L'application Web PolicyEngine US utilise un fork d'OpenFisca pour permettre à quiconque d'estimer ses impôts et ses avantages, et d'estimer l'impact des réformes des politiques personnalisées.
```
Comment (MattiSG) [inline data/showcase/policyengine_us.yml:3]: PolicyEngine US utilise un fork d'OpenFisca pour permettre à quiconque d'estimer ses impôts et aides sociales, ainsi que l'impact de réformes personnalisées.

**Review by MattiSG — CHANGES_REQUESTED** [review]
Thanks a lot @MaxGhenis for this! I made some changes to the French version. I recommend  [DeepL](https://www.deepl.com/translator) over Google Translate, it does an absolutely amazing job 🙂

**Review by MattiSG — CHANGES_REQUESTED** [review]
Waiting for images 🙂 

If possible, please use [ImageOptim](https://imageoptim.com/) to optimise images before committing them to this repo!

File: data/showcase/policyengine_uk.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+title: PolicyEngine UK
+description:
+  fr: PolicyEngine UK utilise un fork d'OpenFisca pour permettre à quiconque d'estimer ses impôts et aides sociales, ainsi que l'impact de réformes personnalisées.
+  en: The PolicyEngine UK web app uses a fork of OpenFisca to let anyone estimate their taxes and benefits, and estimate the impact of custom policy reforms.
```
Comment (MattiSG) [inline data/showcase/policyengine_uk.yml:4]: I am concerned that “fork” is not necessarily clear for the wider audience this website targets. Would something along the lines of the following suggestion work for you? 🙂 
```suggestion
  en: The PolicyEngine UK web app uses an engine derived from OpenFisca to let anyone estimate their taxes and benefits, and estimate the impact of custom policy reforms.
```

**Comment by MattiSG** [comment]
Hi there!

I took the liberty to rebase your branch in order to benefit from all the latest validator improvements that should help you finish referencing PolicyEngine in the showcase 🙂

Could you please take a few minutes to complete the missing information? 😃 In particular, the `source` metadata would be useful here, and illustrations are mandatory.

Automated validation should provide you with feedback as you correct the information.

If you need any help with the information needed, you should be able to find guidance in the [CONTRIBUTING](https://github.com/openfisca/openfisca.org/blob/master/CONTRIBUTING.md#referencing-your-reuse-in-the-showcase) file of this repository. If not, please ask your question here and we'll do our best to help you out (and improve the documentation along the way 😉).

We expect our public website to get significant visibility by next Monday, as OpenFisca will be featured at the World Government Summit. Every reuse that finishes its referencing this week will help demonstrate how useful OpenFisca can be, and will benefit from this added visibility. I hope you can help us make the best of this opportunity! 🙌

**Comment by MaxGhenis** [comment]
Sorry for the delay - I think we're good now

**Review by MattiSG — APPROVED** [review]
Perfect, thanks @MaxGhenis! 🚀

**Pull request merged**